bundle/config/interpolation/interpolation.go

```diff
@@ -9,10 +9,11 @@ import (
 	"sort"
 	"strings"
 
+	"slices"
```

Comment:
Formatting?

Reply:
I think it's not formatted correctly because there's a new line before but technically it's a correct formatting still